src/components/ContactCard/ContactFields/FieldTypes/index.jsx

```diff
@@ -37,8 +37,10 @@ export const Cozy = ({ url }) => {
   )
 }
 
-export const Birthday = ({ value, f }) => {
-  return f(new Date(value), 'YYYY-M-D')
+export const Birthday = ({ value, f, lang }) => {
+  return lang === 'fr'
+    ? f(new Date(value), 'DD-MM-YYYY')
+    : f(new Date(value), 'YYYY-MM-DD')
```

Comment:
Have you considered using `Date.toLocaleDateString()` instead of this ?

Reply:
Either this or maybe the format "P" for the `f` function ? See https://date-fns.org/v2.16.1/docs/format

Reply:
Oui mais ça utilise la langue du système et non la langue configurée dans l'app, ce qui n'est pas la même approche UX. Pour la langue je pense qu'on ne souhaite faire la différence que pour le FR, toutes les autres langues affichent en US. Qu'en penses-tu @poupotte ?

(corrigé pour les '/')

Reply:
Pourquoi ne pas utiliser le format "P" ?

Reply:
Dans Drive, on stocke le format dans la langue.

Et donc on fait 
```
f(updated_at, t('row_update_format'))
```

avec soit : 
    "row_update_format": "D MMM YYYY",
soit
    "row_update_format": "MMM D, YYYY",


Ainsi on supporte plus facilement toutes les langues qu'on ajoute au fur et à mesure

Reply:
Alternative intéressante, ca ressemble à recoder le format "PPP" chez nous par contre non ?

Reply:
Le code est vieux comme le monde à cet endroit, je sais pas si PPP existait déjà, ou si on voulait un affichage différent. Si "P" fait ce qu'on veut ici, autant l'utiliser 👍

Reply:
"P" ne fonctionne pas non `f(new Date(value), 'P')`

Reply:
Ah c'est vrai qu'on est très en retard sur les versions de date-fns, ya pas "P" :/